libnavigation-util/src/test/java/com/mapbox/navigation/utils/thread/ThreadControllerKtTest.kt

```diff
@@ -0,0 +1,58 @@
+package com.mapbox.navigation.utils.thread
+
+import kotlinx.coroutines.channels.Channel
+import kotlinx.coroutines.channels.ClosedReceiveChannelException
+import kotlinx.coroutines.runBlocking
+import org.hamcrest.CoreMatchers.`is`
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertThat
+import org.junit.Assert.assertTrue
+import org.junit.Test
+
+class ThreadControllerKtTest {
```

Comment:
NIT

```suggestion
class ThreadControllerTest {
```

Reply:
I think that name got generated by Android Studio. I see now there was already a class called ThreadControllerTest in the libnavigation-core module, package=com.mapbox.navigation.core however the ThreadController class is in the libnavigation-util module, package=com.mapbox.navigation.utils.thread. I don't have enough context to know if the original test should be moved and consolidated with the test I created or if that test is in its current location for a reason.  I imagine the ThreadController class was moved during some refactoring and the test was left behind but that's speculation on my part.  I renamed the test class. If the two test classes should be consolidated please let me know.